src/node/rpc/test/frontend_test.cpp

```diff
@@ -17,6 +17,14 @@
 #include "node/test/channel_stub.h"
 #include "node_stub.h"
 
+#ifdef PBFT
```

Comment:
It's not clear to me whether this is required? If that's because we need access to `ccf_req`, then I think we can avoid that (see my other comment).